packages/cloud/src/cloud/types.ts

```diff
@@ -0,0 +1,94 @@
+import { Client, ClientOptions } from '@portive/client';
+import { PlateEditor, Value } from '@udecode/plate-core';
+import { ReactEditor } from 'slate-react';
+import { Upload } from '../upload';
+import { createUploadStore } from '../upload/createUploadStore';
+
+/**
+ * Specifies just the `options` part of the CloudPlugin
+ */
+export type CloudPlugin = ClientOptions & {
+  uploadStoreInitialValue?: Record<string, Upload>;
+};
+
+export type CloudEditor<V extends Value = Value> = PlateEditor<V> &
```

Comment:
would rename it to `PlateCloudEditor`
`CloudEditor` would not include `PlateEditor`